core/tests/protractor_desktop/wipeout.js

```diff
@@ -50,10 +53,14 @@ describe('When account is deleted it', function() {
     await users.createAndLoginUser('user1@delete.com', 'userToDelete1');
     await deleteAccountPage.get();
     await deleteAccountPage.requestAccountDeletion('userToDelete1');
+    await waitFor.visibilityOf(
+      pendingAccountDeletionHeading,
+      'Pending Account Deletion Page takes too long to appear');
     expect(await browser.getCurrentUrl()).toEqual(
       'http://localhost:9001/pending-account-deletion');
 
     await users.login('user1@delete.com');
+    await waitFor.pageToFullyLoad();
```

Comment:
Should this be changed to a `waitFor.visibilityOf` too?

Reply:
done